client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -82,10 +84,11 @@
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Set;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 
-import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.CoreMatchers.equalTo;
```

Comment:
`Matchers` is the more idiomatic way to do this.

Reply:
is there any difference? I personally never know which one to pick :)

Reply:
Not really. `Matchers` is the way they document it and the way I've seen it around the internet. `CoreMatchers` is a thing I've seen "organize imports" do from time to time.